src/_bentoml_impl/frameworks/unsloth/template/service.py

```diff
@@ -0,0 +1,200 @@
+from __future__ import annotations
+
+import functools
+import json
+import logging
+import pathlib
+import sys
+import uuid
+from typing import Any
+from typing import AsyncGenerator
+from typing import Literal
+from typing import Optional
+
+import fastapi
+import fastapi.staticfiles
+import pydantic
+import vllm.entrypoints.openai.api_server as vllm_api_server
```

Comment:
This is not needed.

Reply:
Are you suggesting we shouldn't use vllm?

Reply:
On a second thought, using vLLM is fine. Sorry for the confusion.